extension.toml

```diff
@@ -0,0 +1,17 @@
+name = "MATLAB"
+version = "0.0.2"
+authors = ["Ruben Zukic <ruben@zukic.net>"]
+description = "MATLAB syntax highlighting for Zed"
+repository = "https://github.com/rzukic/zed-matlab"
+id = "matlab"
+schema_version = 1
+
+[languages]
+MATLAB = "languages/matlab"
+
+[grammars]
```

Comment:
Have this top-level `grammars` key in addition to `grammars.matlab` is invalid.

We only want the `grammars.matlab` entry.